symfony/plugins/orangehrmTimePlugin/Service/TimesheetService.php

```diff
@@ -563,9 +570,58 @@ public function extractStartDateAndEndDateFromDate(DateTime $date): array
      * @param DateTime $date
      * @return bool
      */
-    public function isTimesheetTakenByDate(DateTime $date)
+    public function hasTimesheetForDate(DateTime $date): bool
     {
         list($startDate) = $this->extractStartDateAndEndDateFromDate($date);
         return $this->getTimesheetDao()->hasTimesheetForStartDate(new DateTime($startDate));
     }
+
+    /**
+     * @param DateTime $startDate
+     * @param int $employeeId
+     * @return Timesheet[]
+     */
+    public function getTimesheetByStartDateAndEmployeeId(int $employeeId, DateTime $startDate): array
+    {
+        return $this->getTimesheetDao()->getTimesheetByStartDateAndEmployeeId($startDate, $employeeId);
+    }
```

Comment:
seems no need this method, since it directly calling DAO

Reply:
done